code/amber/python/lab03b_average_numbers.py

```diff
@@ -0,0 +1,55 @@
+#AVERAGE NUMBERS
+
+from curses.ascii import isalpha
```

Comment:
VSCode will sometimes "help" and import modules for you. In this case, it's likely a module you didn't want. So just keep this in mind and check to ensure it didn't import anything you didn't like.